docs/index.md

```diff
@@ -60,6 +60,7 @@ Components:
 The GitOps Toolkit is always looking for new contributors and there are a multitude of ways to get involved. Depending on what you want to do, some of the following bits might be your first steps:
 
 - Join our upcoming dev meetings ([meeting access and agenda](https://docs.google.com/document/d/1l_M0om0qUEN_NNiGgpqJ2tvsF2iioHkaARDeh6b70B0/view))
+- Ask questions and add suggestions in our [GitOps Toolkit Discussions](https://github.com/fluxcd/toolkit/discussions)
```

Comment:
It's already mentioned two lines below..?

Reply:
Maybe it should be clearer?

Reply:
k. Can we remove this part and change the text below to - Ask questions and add suggestions in our [GitOps Toolkit Discussions](https://github.com/fluxcd/toolkit/discussions)

Below only says "planning discussions" but my understanding is that the GH Discussions is so much more than that and my text was intended to drive more people to the GH Discussions page where can ask questions, get help, _and_ make suggestions / be part of planning. Does that make the intention of this change more clear?

Reply:
That is definitely true per the new Flux community governance https://github.com/scottrigby/flux-community/blob/governance/GOVERNANCE.md#proposal-process.